src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -80,7 +108,7 @@ const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) =>
     const columns = useLiveColumns()
 
     const table = useReactTable({
-        data: markets_data,
+        data: markets_data.get(selected_market) ?? [],
```

Comment:
Can we set a copy of object instead? Sorry for examples, I still don't know what we expecting at the end 😄 
```suggestion
        data: markets_data.get(selected_market) ?? {marked_data1: '', market_data2:''},
```